python/nav/web/static/js/src/portadmin.js

```diff
@@ -84,26 +84,26 @@ require(['libs/spin.min', 'libs/jquery-ui.min'], function (Spinner) {
             return this.addFeedback('Configuring interface ' + $row.find('.port-name').text())
                 .append(this.createProgress());
         },
-        savedInterface: function(listItem, status, message) {
-            status = typeof status === 'undefined' ? 'success' : status;
-            message = typeof message === 'undefined' ? '' : message;
-            listItem.append(this.createAlert(status));
-            if (status !== 'success') {
-                listItem.append($('<small style="margin-left: 1em">').text(message));
-            }
-            listItem.find('progress').remove();
-        },
         restartingInterfaces: function() {
             var restartReason = "<p>A computer connected to a port does not detect that the vlan changes. When that happens the computer will have the IP-address from the old vlan and it will lose connectivity. But if the link goes down and up (a 'restart') the computer will send a request for a new address.</p> 'Restarting' interfaces is only done when changing vlans.";
             var why = $('<span data-tooltip class="has-tip" title="' + restartReason + '">(why?)</span>');
             var listItem = this.addFeedback('Restarting interfaces ').append(why, this.createProgress());
             $(document).foundation('tooltip', 'reflow');
             return listItem;
         },
-        restartInterfacesDone: function(listItem) {
-            listItem.append(this.createAlert('success'));
+        committingConfig: function() {
+            return this.addFeedback('Committing configuration changes')
+                .append(this.createProgress());
+        },
+        endProgress: function(listItem, status, message) {
+            status = typeof status === 'undefined' ? 'success' : status;
+            message = typeof message === 'undefined' ? '' : message;
+            listItem.append(this.createAlert(status));
+            if (status !== 'success') {
+                listItem.append($('<small style="margin-left: 1em">').text(message));
+            }
             listItem.find('progress').remove();
-        }
+        },
```

Comment:
Uh, does this do what you think it does? Not allowed in JSON for instance.

Reply:
I'm no JS expert, but this seems to work in my browser :grin: 

I can easily take it away :)

Reply:
I'd feel safer if you did :) it's supposed to mean there is one more, but undefined item, so it would affect the total length. At least it supposedly did in older javascript.

Reply:
Mm, the diff is misleading anyway - the last element was always superseded by a comma before I touched it :)